src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsComponent.kt

```diff
@@ -12,15 +14,60 @@ import javax.swing.JTextField
  */
 class TestGenieSettingsComponent {
     var panel: JPanel? = null
-    var globalTimeoutTextField = JTextField()
-    var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
+    private var globalTimeoutTextField = JTextField()
+    private var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
+    private var sandboxCheckBox = JCheckBox("Execute tests in a sandbox environment")
+    private var assertionsCheckBox = JCheckBox("Create assertions")
+    private var seedTextField = JTextField()
```

Comment:
Should also be validated?

Reply:
Good observation, what we added is that now when a user inputs an incorrect seed input, they receive a message about it.